src/components/markdown.js

```diff
@@ -30,5 +33,15 @@ export const newWindowLinkRenderer = (href, title, text) => {
 const SimpleMDE = lazy(() => import('react-simplemde-editor'))
 
 export const MarkdownEditor = props => {
-  return h(Suspense, { fallback: centeredSpinner() }, [h(SimpleMDE, props)])
+  return h(Suspense, { fallback: centeredSpinner() }, [h(SimpleMDE, _.merge({
+    options: {
+      autofocus: true,
+      renderingConfig: {
+        singleLineBreaks: false
+      },
+      previewClass: ['editor-preview', 'markdown-body'],
+      previewRender: renderMarkdown,
```

Comment:
Nice! How did you discover this? I don't see it in the docs. :(

Reply:
The docs say that [`options` is passed through](https://github.com/RIP21/react-simplemde-editor#options) to the underlying library, so I [looked through its docs](https://github.com/Ionaru/easy-markdown-editor#configuration)

Reply:
Oh, oops, of course. I was looking at the wrong docs. 🤦